module/Hydro/entry.py

```
@@ -93,21 +93,34 @@ def calculate_ranking(self, submissions: list[SubmissionData]) -> list[RankingDa
         for submission in submissions:
             if submission.at < file_timestamp or submission.verdict != "Accepted":
                 continue
             if (submission.user.uid, submission.problem_id) in problem_ac_list:
                 continue  # 排除同一道题重复ac
             problem_ac_list.append((submission.user.uid, submission.problem_id))
-            for rank in ranking:
-                if submission.user.uid == rank.uid:
-                    rank.accepted = str(int(rank.accepted) + 1)
+            rank = ranking_by_uid.get(submission.user.uid)
+            if rank is None:
+                # 新用户不会被昨日的 ranking 收录，需要根据今日提交补建记录
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)